update_compilers/install_binaries.sh

```diff
@@ -8,32 +8,25 @@ DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
 #########################
 # pahole
-
-TARGET_PAHOLE_VERSION=v1.19
-CURRENT_PAHOLE_VERSION=""
-if [[ -f "${OPT}/pahole/bin/pahole" ]]; then
-    CURRENT_PAHOLE_VERSION=$("${OPT}/pahole/bin/pahole" --version)
-fi
-
-if [[ "$TARGET_PAHOLE_VERSION" != "$CURRENT_PAHOLE_VERSION" ]]; then
+if [[ ! -f "${OPT}/pahole/bin/pahole" ]]; then
     rm -Rf "${OPT}/pahole"
     mkdir -p "${OPT}/pahole"
 
     mkdir -p /tmp/build
     pushd /tmp/build
 
     # Install elfutils for libelf and libdwarf
-    fetch https://sourceware.org/elfutils/ftp/0.182/elfutils-0.182.tar.bz2 | tar jxf -
-    pushd elfutils-0.182
+    ELFUTILS_VERSION=0.188
+    fetch https://sourceware.org/elfutils/ftp/$ELFUTILS_VERSION/elfutils-$ELFUTILS_VERSION.tar.bz2 | tar jxf -
+    pushd elfutils-$ELFUTILS_VERSION
     ./configure --prefix=/opt/compiler-explorer/pahole --program-prefix="eu-" --enable-deterministic-archives --disable-debuginfod --disable-libdebuginfod
```

Comment:
looks like this ought to be updated while you're here to use `${OPT}` like above -oops!

Reply:
thanks for noticing!